File: c/02-condicionais/main022.c
/*Escreva um programa que leia o ano de nascimento de um rapaz e mostre a sua
situação em relação ao alistamento militar.
- Se estiver antes dos 18 anos, mostre em quantos anos faltam para o
alistamento.
- Se já tiver depois dos 18 anos, mostre quantos anos já se passaram do
alistamento.

Data 02/10/2025 -> 10ª Dia
*/

#include <stdio.h>

int main() {

    int ano;
    int alistamento;
    int alta;
    scanf("%d",&ano);

    alistamento = 2025 - ano;
    if(alistamento < 18) {
        printf("Falta %d anos para fazer o alistamento",alistamento);
    }
    else if(alistamento == 18) {

        printf("Voce ja tens a idade para fazer o alistamento!");
    }
    else if(alistamento > 18) {

        alta = alistamento - 18;
        printf("Ja se passaram %d anos para fazer o alistamento",alta);
    }
    return 0;

}

File: python/02-condicionais/main022.py
'''Escreva um programa que leia o ano de nascimento de um rapaz e mostre a sua
situação em relação ao alistamento militar.
- Se estiver antes dos 18 anos, mostre em quantos anos faltam para o
alistamento.
- Se já tiver depois dos 18 anos, mostre quantos anos já se passaram do
alistamento.

Data 02/10/2025 -> 11ª Dia
'''

ano = int(input('Digite o seu ano de nascimento: '))

alistamento = 2025 - ano

if alistamento < 18:
    print(f'Falta {alistamento} anos para o alistamento')

if alistamento == 18:
    print('Você já tens a idade para fazer o listamento')

elif alistamento > 18:
    alta = alistamento - 18
    print(f'Já se passaram {alta} anos para fazer o alistamento')